### DISCUSSION TITLE
Enable multi row taskbar

### DISCUSSION THREAD

**Discussion by derickso** [ramensoftware/windhawk-mods#206] [Mod ideas]
Mod to restore multi-row functionality to the taskbar

**Comment by Anixx** [comment]
Err. What's the problem? It is there still.

**Reply by derickso** [reply]
Where?

**Reply by Anixx** [reply]
If you enable the legacy taskbar, say, via ExplorerPatcher or via registry.

**Reply by derickso** [reply]
That's ExplorerPatcher, not Windhawk

**Reply by Anixx** [reply]
You can enable it via registry https://community.spiceworks.com/topic/2359221-registry-settings-that-help-bring-windows-11-back-to-more-win10-like-experience

**Reply by derickso** [reply]
Are you a developer? What are you arguing with me about? The Windows 10 taskbar is not Windhawk
